### PR TITLE
Remove layerCreator for logical ops.

### PR DESCRIPTION
**Pull request by sdurawa**
Remove layerCreator for:
- LogicalNot
- LogicalAnd
- LogicalOr
- LogicalXor
- LessEqual

Ticket: 43476

### CODE REVIEW

**Review by blesniewski — APPROVED** [review]
LGTM

File: ngraph/core/src/op/or.cpp [hunk 1]
```diff
@@ -34,6 +34,13 @@ op::v1::LogicalOr::LogicalOr(const Output<Node>& arg0,
     constructor_validate_and_infer_types();
 }
 
+bool ngraph::op::v1::LogicalOr::visit_attributes(AttributeVisitor& visitor)
+{
+    NGRAPH_OP_SCOPE(v1_LogicalOr_visit_attributes);
+    BinaryElementwiseLogical::visit_attributes(visitor);
```
Comment (GlebKazantaev) [inline ngraph/core/src/op/or.cpp:40]: Why we need to override visit_attributes if it is already implemented in base class?

**Review by GlebKazantaev — APPROVED** [review]

File: ngraph/core/src/op/or.cpp (hunk 1 above)
Comment (ilyachur) [reply ngraph/core/src/op/or.cpp:40]: I propose to fix it in this PR.

Comment (sdurawa) [reply ngraph/core/src/op/or.cpp]: Corrected.

**Review by ilyachur — APPROVED** [review]

**Pull request merged**